### ISSUE TITLE
Rounded corners and border on Windows 11

### ISSUE THREAD

**Issue by heksesang** [lapce/lapce#3578]
## Is your feature request related to a problem? Please describe.

Currently on Windows 11, if Lapce is not in maximized mode, the main window of the application has sharp corners, and it looks like the window lacks a border and drop shadows.

The lack of rounded corners doesn't match other apps on my system and doesn't really match the design language of Lapce itself which uses rounded corners in the UI.

The lack of border/drop shadow can make it hard to distinguish from other applications with similar or same color scheme.

## Describe the solution you'd like

Add a proper border and drop shadows around the window and make the window use rounded corners.

**Comment by Long0x0** [comment]
I created a PR at [lapce/floem](https://github.com/lapce/floem/pull/643).

**Comment by Long0x0** [comment]
Yeah, once that’s in place, we’ll be able to apply it by adding `undecorated_shadow` to [default_window_config](https://github.com/lapce/lapce/blob/6ef86351fc9574fe739ddb73ec1940ec2b479c54/lapce-app/src/app.rs#L199).
